extensionsGallery-insider.json

```diff
@@ -1511,7 +1511,7 @@
 								},
 								{
 									"key": "Microsoft.AzDataEngine",
-									"value": ">=1.40.0"
+									"value": ">=1.44.0"
```

Comment:
@SakshiS-harma you missed updating the azdataengine for sql-database-project extension in this PR, but updated in the other one, could you please make them consistent?

Reply:
I updated insiders a couple of weeks back for projects, so the updated azdataengine version is in there already.